src/Workleap.DomainEventPropagation.sln

```diff
@@ -65,8 +65,10 @@ Global
 		{B2F1E4B9-C32E-4835-BD1A-557ACCA85B58}.Release|Any CPU.Build.0 = Release|Any CPU
 		{20771920-AB99-49F0-92A9-7D341B156F91}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{20771920-AB99-49F0-92A9-7D341B156F91}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{20771920-AB99-49F0-92A9-7D341B156F91}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{8883071A-7B43-4B4C-9647-BE6CB0F16457}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{8883071A-7B43-4B4C-9647-BE6CB0F16457}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{8883071A-7B43-4B4C-9647-BE6CB0F16457}.Release|Any CPU.ActiveCfg = Release|Any CPU
```

Comment:
There's still a line missing for each item, if you look each guid has 4 items and these two only have 3. What happnes is it fails to build the solution in Release in your IDE because the analyzer project won't be built

```suggestion
		{20771920-AB99-49F0-92A9-7D341B156F91}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
		{20771920-AB99-49F0-92A9-7D341B156F91}.Debug|Any CPU.Build.0 = Debug|Any CPU
		{20771920-AB99-49F0-92A9-7D341B156F91}.Release|Any CPU.ActiveCfg = Release|Any CPU
		{20771920-AB99-49F0-92A9-7D341B156F91}.Release|Any CPU.Build.0 = Release|Any CPU
		{8883071A-7B43-4B4C-9647-BE6CB0F16457}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
		{8883071A-7B43-4B4C-9647-BE6CB0F16457}.Debug|Any CPU.Build.0 = Debug|Any CPU
		{8883071A-7B43-4B4C-9647-BE6CB0F16457}.Release|Any CPU.ActiveCfg = Release|Any CPU
		{8883071A-7B43-4B4C-9647-BE6CB0F16457}.Release|Any CPU.Build.0 = Release|Any CPU
```